tests/test_initializer.py

```diff
@@ -111,6 +115,38 @@ def test_initialize_valid_domain(domains_config, initializer_metrics):
     assert [','.join(o['labels']) for o in initializer_metrics.observations] == [',success', ',initialized', ',success', ',initialized']
 
 
+def test_volume_config_hostname_worker_id_cache(domains_config, initializer_metrics):
```

Comment:
the change is in domains_config get_cwm_api_volume_config, so the test should test that function and be in test_domains_config